bundles/org.openhab.binding.dominoswiss/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="dominoswiss" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Dominoswiss Binding</name>
+	<description>The Dominoswiss Binding interacts with the Dominoswiss eGate G1 Gateway to control blinds</description>
+	<author>Frieso Aeschbacher</author>
```

Comment:
The author tag is deprecated and should therefore be removed. See openhab/openhab-core#1844.